dev-tools/install.sh

```diff
@@ -72,6 +72,7 @@ cd $(dirname "$BASH_SOURCE")/..
 if [ "$1" == "--clean" ]; then
   echo "Removing installed dependencies and compiled static files."
   rm -rf .venv node_modules src/cms/static
+  python3 -m venv .venv
```

Comment:
Pipenv should automatically create the `.venv` directory in case it doesn't exist... maybe we need another `--pipenv-in-venv` parameter or similar for people who want to install pipenv inside the venv and not via their package manager or `pip --user`...

Reply:
This seems to be a left over from previous experiments I did.